### ISSUE TITLE
You can add the option to rename the VPK? 

### ISSUE THREAD

**Issue by nemesiscbm** [soywiz/vitaorganizer#6]
add a command to rename the vpk with the name of the game.

example PCSE00790_01.00.VPK to BASTION_PCSE00790_01.00.VPK

thanks.

**Comment by soywiz** [comment]
Sure, makes sense.

**Comment by nemesiscbm** [comment]
Thanks,  gracias ;) ( que veo que eres español ).

**Comment by sickness0666** [comment]
i think a better option would be to be able to make our own naming schematic with keywords such as:

[#gameid#] #title# #version# [#region#]
[#gameid#]_#title#_#version#_#region#

or something similar so that we can specify how we want the files renamed, have the available keywords be things that can be extracted from the .sfo file, i dont know what all can be extracted

**Comment by soywiz** [comment]
Yeah, I thought that. Probably expose SFO keys and allow to customize. And allow to rename all games at once, so it gets sync.

**Comment by soywiz** [comment]
I noticed that @luckylucks fork added some work related to this:
https://github.com/luckylucks/vitaorganizer/commit/9444e41be09eb9d979f2a34a0c53246db3bfdaf1

Not sure the status, but looks great.
